mods/parsoid.js

```diff
@@ -459,7 +460,27 @@ PSP.transformRevision = function (restbase, req, from, to) {
         var body2 = {
             original: original
         };
-        body2[from] = req.body[from];
+        if (from === 'sections') {
+            try {
+                var sections = JSON.parse(req.body.sections);
```

Comment:
@d00rman Actually, this endpoint is made in analogy to other transformation endpoints, so they accept a form-data. So, in case application/json header is defined it will fail way before this point with 400 'Missing request parameter sections'. So, I suppose it's enough to add .toString() and check that we've got an object after parsing to protect against smth like sections=1234 where JSON.parse wouldn't throw an exception. What do you think?